### PR TITLE
SDD-1002 - UI chanages to course publish

### PR DESCRIPTION
**Pull request by MrKevJoy**
- Changed Course name to course.
- Correct page title that mentioned Venue.
- Added LearnAimRefTitle to table.

### CODE REVIEW

File: src/Dfc.CourseDirectory.WebV2/Features/DataManagement/Courses/CheckAndPublish.cs [hunk 1]
```diff
@@ -106,15 +112,19 @@ public class Handler :
...
+
+            return new ViewModel()
             {
                 RowGroups = rows
                     .GroupBy(t => t.CourseId)
                     .Select(g => new ViewModelRowGroup()
                     {
                         CourseId = g.Key,
                         LearnAimRef = g.Select(r => r.LearnAimRef).Distinct().Single(),
+                        LearnAimRefTitle = learningDelivery[g.Select(r => r.LearnAimRef).Distinct().Single()]?.LearnAimRefTitle,
```
Comment (gunndabad) [inline src/Dfc.CourseDirectory.WebV2/Features/DataManagement/Courses/CheckAndPublish.cs:127]:
```suggestion
                        LearnAimRefTitle = learningDelivery[g.Select(r => r.LearnAimRef).Distinct().Single()].LearnAimRefTitle,
```

File: src/Dfc.CourseDirectory.WebV2/Features/DataManagement/Courses/CheckAndPublish.cshtml [hunk 2]
```diff
@@ -35,14 +35,19 @@
...
-                                            <th scope="col" class="govuk-table__header">Course name</th>
+                                            <th scope="col" class="govuk-table__header">Course</th>
                                             <th scope="col" class="govuk-table__header" style="width: 150px">Delivery type</th>
                                             <th scope="col" class="govuk-table__header" style="width: 130px">Start date</th>
                                         </tr>
                                     </thead>
                                     <tbody class="govuk-table__body">
                                         @foreach (var row in Model.RowGroups)
                                         {
+                                            <tr>
```
Comment (gunndabad) [inline src/Dfc.CourseDirectory.WebV2/Features/DataManagement/Courses/CheckAndPublish.cshtml:46]: This is different to the other 4 places where we've added it. Is that intention?

Comment (gunndabad) [reply src/Dfc.CourseDirectory.WebV2/Features/DataManagement/Courses/CheckAndPublish.cshtml:46]: example: https://github.com/SkillsFundingAgency/dfc-coursedirectory/pull/2154/files#diff-5e790bcfda982e2ded6fca2df647db03f838ff3f501f4135760b098d5817ccd9R29-R32

Comment (gunndabad) [reply src/Dfc.CourseDirectory.WebV2/Features/DataManagement/Courses/CheckAndPublish.cshtml:46]: Possibly need the right-aligned date too: https://github.com/SkillsFundingAgency/dfc-coursedirectory/pull/2154/files#diff-5e790bcfda982e2ded6fca2df647db03f838ff3f501f4135760b098d5817ccd9R65

Comment (MrKevJoy) [reply src/Dfc.CourseDirectory.WebV2/Features/DataManagement/Courses/CheckAndPublish.cshtml]: oversight on my behalf.

File: src/Dfc.CourseDirectory.WebV2/Features/DataManagement/Courses/CheckAndPublish.cs (hunk 1 above)
Comment (MrKevJoy) [reply src/Dfc.CourseDirectory.WebV2/Features/DataManagement/Courses/CheckAndPublish.cs]: done

**Review by gunndabad — APPROVED** [review]

**Pull request merged**